src/Pact/Types/Gas.hs

```diff
@@ -222,10 +224,22 @@ instance FromJSON GasLimit where
 
 instance Wrapped GasLimit
 
+-- Todo: this probably overflows but do we care?
+gasLimitToMilliGasLimit :: GasLimit -> MilliGasLimit
+gasLimitToMilliGasLimit (GasLimit (ParsedInteger i)) =
+  MilliGasLimit (gasToMilliGas (fromIntegral i))
+
+newtype MilliGasLimit
+  = MilliGasLimit MilliGas
+  deriving (Eq, Ord)
+
+instance Show MilliGasLimit where
```

Comment:
Why not use the derived `Show` instance? It carries more info about the units.

Reply:
I'm afraid of it making it into any outputs.